datastore/h2gis/src/main/groovy/org/orbisgis/datastore/h2gis/H2GIS.groovy

```diff
@@ -51,6 +51,20 @@ import groovy.transform.Field
 /** Cached H2GISDataStoreFactory. */
 private static final @Field H2GISDataStoreFactory H2GIS_DATA_STORE_FACTORY = new H2GISDataStoreFactory()
 
+
+/**
+ * Open a H2GIS DataStore
+ * @param databasePath a path to the database
+ * @return An H2GIS DataStore.
+ */
+static JDBCDataStore open(String databasePath) {
+    Map params = new java.util.HashMap()
+    params.put("database", new File(databasePath).absolutePath)
+    params.put("dbtype", "h2gis")
+    return H2GIS_DATA_STORE_FACTORY.createDataStore(params)
```

Comment:
Can it be replaced by `return H2GIS_DATA_STORE_FACTORY.createDataStore([database:new File(databasePath).absolutePath])` ?